go.mod

```diff
@@ -8,8 +8,8 @@ require (
 	github.com/PuerkitoBio/purell v1.2.0
 	github.com/argoproj/argo-rollouts v1.4.1
 	github.com/cenkalti/backoff/v4 v4.2.1
-	github.com/dapr/components-contrib v1.11.0-rc.11
-	github.com/dapr/kit v0.11.2
+	github.com/dapr/components-contrib v1.11.3-0.20230621082614-31485cc1e772
```

Comment:
Updates components-contrib to fix an issue with Azure Key Vault (which impacts subtle APIs only)